Topics/Tech_Stacks/Metaprogramming.md

```diff
@@ -0,0 +1,142 @@
+### goal
```

Comment:
More descriptive title would be beneficial

Reply:
I changed it from goal to **what we are going to cove**